src/main/java/service/impl/FileWriterImpl.java

```
@@ -0,0 +1,17 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import service.FileWriter;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String fileName) {
```

The method does not check for null or empty parameters (data or fileName). According to the checklist, you should validate parameters and throw a RuntimeException if invalid (e.g., null or empty fileName or data).